src/provider/provider_level_zero.c

```diff
@@ -266,6 +309,19 @@ static void init_ze_global_state(void) {
         utils_close_library(lib_handle);
         return;
     }
+
+    if (ze_init_drivers(lib_handle, lib_name) != UMF_RESULT_SUCCESS) {
+        LOG_INFO("Initializing Level Zero through zeInitDrivers failed. "
+                 "Falling back to zeInit.");
```

Comment:
Should we fall back to the `zeInit` only if `zeInitDrivers` is unavailable?
E.g. should we call `zeInit` if `zeInitDrivers` is available but returns some error?